makefile

```
@@ -0,0 +1,50 @@
+.PHONY: build bash run run-dev stop down logs tests help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)